mantidproject/mslice
# Changes to cuts for new interface

**Not to be merged until after #209**

- Add some workspace type specific controls to `main_window`
- Each cut is now saved as a workspace
- Some refactoring to `mslice.models.cut` to allow `MDHistoWorkspaces` to be loaded / plotted independently. Before, the cut parameters and parent workspace were cached and re-run every time the cut was plotted. Now, the data is saved and read from the `MDHistoWorkspace`.
- Changed name of cuts that are saved to a file to include integration limits -  before, they looked like `MAR21335_Ei60.00meV_QE_cut1` with an incrementing index on the end. Now they look like this: `MAR21335_Ei60_meV_QE_cut(-1.0,1.0) `
- Automatically change tabs after load/powder/plot cut operations

**To test:**
- Test all buttons that appear on each workspace tab function as expected
- save a cut to a file, then load it in a different session and plot (without a need for the parent workspace)

Fixes #51 
Fixes #220
